lessons/23-js-logical-operators/index.js

```diff
@@ -0,0 +1,10 @@
+let currentHour = prompt('enter current hour 0-24', '');
+if (currentHour < 0 || currentHour>23) {
+  alert( `${currentHour} is incorrect input`);
+} 
+if (currentHour<8 || currentHour>=18) {
+    alert( `${currentHour} is outside of office hour`);
+} 
+if(currentHour>=8 && currentHour<18) {
+    alert( `We are open`);
+}
```

Comment:
Only suggestion is that since these are all related, we could have used `if/else if` here instead of 3 standalone `if` statements, but either way is correct 😄